meshapp/src/components/Footer.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import {
+  Grid,
+  Link,
+  Stack,
+  ThemeProvider,
+  Typography,
+  createTheme,
+} from "@mui/material";
+document.body.style.color = "black";
+
+const Footer = () => {
+  let theme = createTheme({
+    components: {
+      MuiLink: {
+        styleOverrides: {
+          root: {
+            textDecoration: "none",
+            color: "black",
+            fontSize: "11px",
+            fontWeight: "600",
+            "&:hover": {
+              color: "#1db272",
+              transition: "200ms ease-in",
+            },
+          },
+        },
+      },
+    },
+    typography: {
+      h1: {
+        fontWeight: "600",
+        fontSize: "20px",
+        lineHeight: "20px",
+      },
+      h2: {
+        textDecoration: "underline",
+        fontSize: "13px",
+        fontWeight: "600",
+      },
+      h3: {
+        fontSize: "13px",
+        fontWeight: "400",
+      },
+    },
+  });
+
+  return (
+    <ThemeProvider theme={theme}>
+      {/*footer*/}
+      <Grid container direction="column" xs={12} sx={{ background: "#F2E8DE" }}>
+        {/*footer logo and links container*/}
+        <Grid
+          container
+          display="flex"
+          direction="row"
+          justifyContent="space-between"
+          rowGap={2}
+          xs={12}
+          p={2}
+        >
+          <Grid item>
+            <Typography color="#1DB272" variant="h1">
+              Mesh
+            </Typography>
+          </Grid>
+
+          <Grid
+            container
+            direction="row"
+            justifyContent="space-around"
+            xs={11}
+            sm={6}
+            sx={{ flexWrap: "nowrap" }}
+          >
+            {/**
+             * @TODO : add actual links to the link elements (https://github.com/LetsMesh/Site/issues/191)
```

Comment:
Create another issue / task for this so it isn't lost in code :D (put the reference of the TODO in the issue description)

Reply:
Isn't this the issue I linked in the TODO? what does the reference of the TODO exactly mean?